src/net/network_manager.cpp

```diff
@@ -45,6 +73,22 @@ void ServerNetworkManager::update() {
                 events::build::try_build_event(construct);
                 break;
             }
+            case proto::ClientMessage::MessageTypeCase::kMoveRequest: {
+                proto::StickData stick = msg.move_request();
+                if (stick.input() == proto::StickData_Stick::StickData_Stick_STICK_LEFT)
```

Comment:
I think you can use `proto::StickData::STICK_LEFT` to reduce verbosity. Ditto for other instances of this.